tests/foreman/cli/test_errata.py

```diff
@@ -1437,3 +1435,59 @@ def test_update_applicable_package_using_default_content_view(chost):
         }
     )
     assert len(applicable_packages) == 0
+
+
+@pytest.mark.tier2
+def test_downgrade_applicable_package_using_default_content_view(chost):
+    """Downgrading a package on a host attached to the default content view
+    causes the package to become applicable and installable.
+
+    :id: 8503dff8-c2d9-4818-a607-746dc551894b
+
+    :steps:
+        1. Register a host that already requires errata to org with Library
+        2. Update the aplicable package
+        3. Ensure the expected package is not applicable on the newly registered host
+        4. Downgrade the applicable package on the host
+        5. Ensure the package is now applicable
+
+    :expectedresults: downgraded package now shows as applicable and installable
```

Comment:
expected results can/should be a list that aligns with `:steps:`. So expected result 1 should be what you expect to validate or assert after step 1 is complete.

Reply:
I will make another PR